src/cms/src/api/protected-area/content-types/protected-area/schema.json

```diff
@@ -367,6 +367,12 @@
       "relation": "oneToMany",
       "target": "api::park-date.park-date",
       "mappedBy": "protectedArea"
+    },
+    "parkGate": {
+      "type": "relation",
+      "relation": "oneToOne",
+      "target": "api::park-gate.park-gate",
+      "inversedBy": "protectedArea"
     }
```

Comment:
Probably `"mappedBy": "protectedArea"`?

Reply:
The code was generated by the Strapi Content-Type Builder and it seems to work as expected